config/environments/production.rb

```diff
@@ -71,6 +75,18 @@
   # to allow testing with docker-compose-local-prod.yml
   config.hosts << "localhost"
 
+  # production and staging urls
+  config.hosts << "apps.concord.org"
+  config.hosts << "apps.concordqa.org"
+  config.hosts << "vpb.concord.org"
+  config.hosts << "vpb.concordqa.org"
```

Comment:
At least with LARA and the Portal we specify the host name as a CloudFormation template parameter. And we then add it is an environment variable in the app task description. If the same thing was done here then this hosts array could be setup from that environment variable. 

Since it would make this easier I'd vote to just serve it from vpb.concord.org instead of from both apps.concord.org/vpb and from vpb.cocnord.org